examples/offloader.js

```diff
@@ -12,17 +14,17 @@ var fs = require('fs');
 
 var client = new SteamUser();
 var manager = new TradeOfferManager({
-	"steam": client, // Polling every 30 seconds is fine since we get notifications from Steam
```

Comment:
It's still more valid than if it didn't have the quotes. I find it helpful to get in one habit of writing JSON(-like) objects. Double quotes around keys and values, and no dangling commas.